x-pack/plugins/infra/server/lib/host_details/process_list.ts

```diff
@@ -25,6 +25,7 @@ export const getProcessList = async (
               [timefield]: {
                 gte: to - 60 * 1000, // 1 minute
                 lte: to,
+                format: 'epoch_millis',
```

Comment:
This is called when fetching the process list of a node in the Metrics waffle chart